jupyter-book/introduction/analysis_tools.ipynb

```diff
@@ -52,9 +52,16 @@
    "cell_type": "markdown",
```

Comment:
<p>Would be nice to link to the youtube channel as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='116'/>